learn/app-development/deployment/build-with-docker.md

```diff
@@ -176,10 +170,10 @@ docker container run -d --name <container-name> -v <project-location>/dist/:/usr
 example: docker container run -d --name wm-app -v /home/user/Downloads/myapplication/dist/:/usr/local/tomcat/webapps/ -p 80:8080 tomcat:8.5
 ```
 
```

Comment:
Add an Headding :: Access Application

write some info like if you configured host network for docker only